libfuse/file.go

```diff
@@ -11,19 +11,41 @@ import (
 	"golang.org/x/net/context"
 )
 
+type eiCache struct {
+	ei  libkbfs.EntryInfo
+	rid string
+}
+
 // File represents KBFS files.
 type File struct {
 	folder *Folder
 	node   libkbfs.Node
+
+	// for the Attr call after Create
+	eiCache *eiCache
```

Comment:
It might make sense to set this to `nil` on every modifying operation, including invalidates if that's easy.  Just in case another operation slips in between the create and the attr call. (In which case, you'd definitely need a lock.)